### PR TITLE
added TN and OH

### PR DESCRIPTION
**Pull request by SteveWallace**

### CODE REVIEW

File: scrapers.js [hunk 1]
```diff
@@ -1148,6 +1148,53 @@ let scrapers = [
...
+      let counties = [];
+      let $ = await fetch.page(this.url);
+      let $paragraph = $('p:contains("Number of counties with cases:")').text();
+      let regExp = /\(([^)]+)\)/;
+      let parsed = regExp.exec($paragraph);
+      let arrayOfCounties = parsed[1].split(',');
+      arrayOfCounties.map(county => {
+        let splitCounty = county.trim().split(' ');
+        counties.push({
+          county: splitCounty[0] + ' County',
```
Comment (lazd) [inline scrapers.js]: throw a `parse.string(splitCounty[0])` in to clean out whitespace, whack random zero-width chars, etc.

File: scrapers.js [hunk 2]
```diff
@@ -1148,6 +1148,53 @@ let scrapers = [
...
+          cases: splitCounty[1]
```
Comment (lazd) [inline scrapers.js]: Throw a `parse.number()` around this to make it a real number.

Comment (SteveWallace) [reply scrapers.js]: added

File: scrapers.js (hunk 1 above)
Comment (SteveWallace) [reply scrapers.js]: Added

**Comment by lazd** [comment]
Nice work, thank you!

**Pull request merged**